.vscode/cspell.json

```diff
@@ -1228,6 +1228,12 @@
         "runpath",
         "Cves"
       ]
+    },
+    {
+      "filename": "**/sdk/monitor/**/*.cs",
+      "words": [
+        "resourceid"
```

Comment:
Is this still needed?

Reply:
https://github.com/Azure/azure-sdk-for-net/issues/39340

Reply:
will be resolved before GA